index.js

```diff
@@ -113,7 +113,11 @@ exports.fromMapFileComment = function (comment, dir) {
 
 // Finds last sourcemap comment in file or returns null if none was found
 exports.fromSource = function (content, largeSource) {
-  if (largeSource) return convertFromLargeSource(content);
+  if (largeSource) {
+    var res = convertFromLargeSource(content);
+    if (typeof res === 'undefined') res = null;
+    return res;
```

Comment:
Ha yeah - for some reason I was feeling like testing for undefined explicitly.  I'll change that and update the pr tonight.